nilearn/decoding/tests/test_decoder.py

```diff
@@ -82,7 +93,7 @@
         extra_valid_checks=extra_valid_checks,
     ),
 )
-def test_check_estimator_decoder_regressor(estimator, check, name):  # noqa: ARG001
+def test_check_estimator_decoder_regressor(estimator, check, name):
```

Comment:
These `# noqa: ARG001` removals are unrelated but we had too many of them. So I just excluded the whole file for this ruff check.